composer.json

```diff
@@ -39,5 +40,8 @@
         "branch-alias": {
             "dev-master": "1.x-dev"
         }
+    },
+    "scripts": {
+        "test": "@php vendor/bin/phpunit"
```

Comment:
can you please also change th .github/workflows/ci.yml file to run `composer test` instead of calling phpunit directly? that way we make sure that this line actually works.